File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    int hashtags = 1;
    int spacechange = height;

    for (int i = 0; i < height; i++)
    {
        int spaces = spacechange - 1;
        // To print spaces
        for (int j = 0; j < spaces; j++)
        {
            printf(" ");
        }
        // print the hashtags on the left side
        for (int k = 0; k < hashtags; k++)
        {
            printf("#");
        }
        // print the two spaces between the hashtags
        for (int l = 0; l < 1; l++)
        {
            printf("  ");
        }
        // print the hashtags on the right side
        for (int m = 0; m < hashtags; m++)
        {
            printf("#");
        }

        printf("\n");
        hashtags++;
        spacechange--;
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height >= 1 and height <= 8:
        break

num_of_space = height - 1
hashtag_count = 1
for i in range(height):
    print(" " * num_of_space, end="")
    print("#" * hashtag_count, end="")
    print("  ", end="")
    print("#" * hashtag_count)
    num_of_space = num_of_space - 1
    hashtag_count = hashtag_count + 1
